### PR TITLE
Add support to use existing repo for IIS hosted WCF test service

### LINKED ISSUE #1085: Create a script to setup the developer's machine to run IISHostedWcfService in IIS
Create a simplified version of tools\scripts\WcfTestServerSetup.cmd to configure IIS to be able to run the IISHostedWcfService in IIS on a developer's machine using their current enlistment.

The goal is to make this process easier for developers and to minimize the need for error-prone manual steps.

### PR DESCRIPTION
**Pull request by zhenlan**
Usage:
   /p: If an existing WCF repo is preferred to be used, use this parameter to provide
       the path to a WCF repo.

Examples:
   SetupWcfIISHostedService 42 /p:"c:\my\existing\wcf\repo"
   :Create IIS hosted service named 'wcfService42' by using existing WCF repo located at "c:\my\existing\wcf\repo"

fixes #1085

### CODE REVIEW

File: src/System.Private.ServiceModel/tools/scripts/SetupWcfIISHostedService.cmd [hunk 1]
```diff
@@ -169,18 +203,22 @@ goto :Done
...
 echo.
-echo Usage: %~n0 Id [/c]
+echo Usage: %~n0 Id [/c] [/p:{PathToWcfRepo}]
 echo    Id: The Id of a WCF repo and its associated IIS hosted services to be created.
 echo        The Id will be appended to the name of all assets to be created.
 echo        %_idMaster%: if Id is '%_idMaster%', additional IIS hosted services such as PRService
 echo        will be created to serve as central services for other WCF repos. 
 echo    /c: If specified, this will clean up any existing setup of Id instead of creating new.
+echo    /p: If an existing WCF repo is preferred to be used, use this parameter to provide
+echo        the path to a WCF repo.
```
Comment (roncain) [inline src/System.Private.ServiceModel/tools/scripts/SetupWcfIISHostedService.cmd]: Nit: optionally add to this explanation what will happen if /p is not provided.  In other words, what is the default?

**Comment by roncain** [comment]
LGTM
I made one small non-blocking comment

**Comment by iamjasonp** [comment]
NIT: One of the things that's weird to me is that we specify "42" and that implies a service with endpoint wcfService42, but that no longer has a bearing to something on the filesystem (e.g., repo at wcf42). I think an interesting change might be the ability to change the parameter to use something like 

SetupWcfIISHostedService **wcfService**42 /p:"c:\my\existing\wcf\repo" 

So that it removes this inbuilt assumption that you want your endpoint service to be prefixed with "wcfService" 

Not blocking

**Comment by iamjasonp** [comment]
LGTM

File: src/System.Private.ServiceModel/tools/scripts/SetupWcfIISHostedService.cmd (hunk 1 above)
Comment (zhenlan) [inline src/System.Private.ServiceModel/tools/scripts/SetupWcfIISHostedService.cmd]: fixed

**Pull request merged**